test/unleash.network.retries.test.js

```diff
@@ -0,0 +1,39 @@
+import test from 'ava';
+import { createServer } from 'http';
+import { Unleash } from '../lib/unleash';
+
+test.cb('should retry on error', (t) => {
+  t.plan(1);
+
+  let calls = 0;
+  const server = createServer((req, res) => {
+    calls ++;
```

Comment:
```suggestion
    calls++;
```

weird space, might just be me 😀